packages/react/src/core/richtext-hoc.ts

```diff
@@ -42,6 +48,8 @@ export function createRichTextHook(
     // Use the same component resolver that's exported
     const componentResolver = createComponentResolver(Component, { isServerContext });
 
+    const { resolvers: _, ...optionsWithoutResolvers } = options;
```

Comment:
question(non-blocking): I don't understand this. Maybe we can clarify the code somehow to make it self explanatory why this is necessary?